app/views/join_request_mailer/join_request.text.erb

```diff
@@ -5,7 +5,7 @@ If you would like to be removed from the mailing list or Slack, let us know by r
 
 To learn more about AB Tech and what we do:
 
-1. Check out the rest of our website at https://www.abtech.org and follow us on Instagram @cmuabtech.
+1. Check out the rest of our website at https://www.abtech.org and follow us on Instagram [@cmuabtech](https://www.instagram.com/cmuabtech/).
```

Comment:
Might be good to add an HTML template along with it if adding a link. See other mailers for an example.